legacy/builder/utils/utils.go

```diff
@@ -189,22 +148,11 @@ func TrimSpace(value string) string {
 }
 
 func PrepareCommand(pattern string) (*exec.Cmd, error) {
-	parts, err := ParseCommandLine(pattern)
+	parts, err := properties.SplitQuotedString(pattern, `"'`, false)
```

Comment:
Would this change the behaviour of the `compile` command `--build-property` and `--build-properties` flags?

Reply:
No it should work exactly the same. This is code duplication that has never been removed.
(BTW a quick test would not be bad :-))